### PR TITLE
Fixed Intermittent failure of ItElasticLogging and ItElasticLoggingFluentd 

### PR DESCRIPTION
**Pull request by hzhao-github**
Not seeing the failures  in 8 jobs from elk_nightly
 
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9630
to
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9638

with latest changes, the failures  are not seen in 10 jobs from elk_nightly
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9767
to
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9776

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java [hunk 1]
```diff
@@ -280,11 +294,13 @@ void testFluentdQuery() {
     }
 
     logger.info("Total count of logs: " + count);
-    assertTrue(count > 0, "Total count of logs should be more than 0!");
-    assertEquals(0, failedCount, "Total failed count should be 0!");
     logger.info("Total failed count: " + failedCount);
 
-    logger.info("Query logs of serverName={0} succeeded", adminServerPodName);
+    if (count > 0 && failedCount == 0) {
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java:299]: You can simplify this to just return the test: `return count > 0 && failedCount == 0;`

**Review by anpanigr — COMMENTED** [review]
The Jenkin Results are not accessible. Please re run the tests with latest sync to main branch

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java (hunk 1 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java]: changed

**Review by rjeberhard — APPROVED** [review]

**Review by anpanigr — APPROVED** [review]

**Comment by anpanigr** [comment]
> @anpanigr Jenkins job is passing at https://build.weblogick8s.org:8443/job/WebLogic%20Kubernetes%20Operator%20Kind/view/change-requests/job/PR-2903/

I was looking for complete kind-parallel run, as this test is not part of gate

**Pull request merged**